plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -76,17 +77,20 @@ func Run(s *options.SchedulerServer) error {
 	// cache only non-terminal pods
 	podInformer := factory.NewPodInformer(kubecli, 0)
 
+	var configMapInformer cache.SharedIndexInformer
+	if !s.UseLegacyPolicyConfig && len(s.PolicyConfigMapName) != 0 {
+		// cache only scheduler's policy ConfigMap
+		configMapInformer = factory.NewPolicyConfigMapInformer(kubecli, 0, s.PolicyConfigMapName, s.PolicyConfigMapNamespace)
+	} else {
+		configMapInformer = nil
+	}
+
 	sched, err := CreateScheduler(
 		s,
 		kubecli,
-		informerFactory.Core().V1().Nodes(),
+		informerFactory,
```

Comment:
giving the scheduler access to all informers for all objects seem like an antipattern. enumerating the objects it cares about lets us bound the inputs to the scheduler